core/arch/arm/arm.mk

```diff
@@ -113,6 +113,7 @@ $(call force,CFG_CORE_FFA,y)
 $(call force,CFG_CORE_SEL1_SPMC,n)
 $(call force,CFG_CORE_EL3_SPMC,n)
 CFG_CORE_HAFNIUM_INTC ?= y
+CFG_CORE_RELOCATABLE ?= y
```

Comment:
I think this flag deserves some description here, or maybe a different name, because with ASLR the core is already relocatable in the VA space, but that's a different relocation we're dealing with here. How about `CFG_CORE_PHYS_RELOCATABLE`?

Reply:
Maybe move description in _mk/config.mk_?
```make
# Enable support in OP-TEE to relocate itself to allow it to run from a
# physical address that differs from the link address. This feature is
# currently supported only when CFG_CORE_SEL2_SPMC is enabled.
CFG_CORE_RELOCATABLE ?= n
$(eval $(call cfg-depends-one,CFG_CORE_RELOCATABLE,CFG_CORE_SEL2_SPMC))
```

Reply:
I'm not sure that's the best place. With `CFG_CORE_SEL2_SPMC` we should use `y` as the default value, else `n`. This is likely to change as more and more configurations can support `CFG_CORE_RELOCATABLE`. The number of configuration options enabled with `CFG_CORE_SEL2_SPMC=y` is also expected to grow and it's not nice to have that in too many different places.

Reply:
I think it is legitimate for _arm.mk_ to default enable the config and for _mk/config.mk_ to default disable that config.
The only point is that there should be only one place where the config switch description can be found.

That said, as currnetly only SPM'ed OP-TEE can support this feature, I'm fine having the config and inline description as currently proposed by this P-R.

Reply:
OK, thanks.